enhancements/sig-architecture/20-taint-toleration/README.md

```diff
@@ -0,0 +1,312 @@
+# Taint-Toleration in Placement APIs
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The proposed work would define Taints and Tolerations in placement APIs to allow users to control the selection of managed clusters more flexibly, and evict selected unhealthy/not-reporting clusters after a certain period.
+
+## Motivation
+
+For the new placement enhancement need of supporting filtering unhealthy/not-reporting clusters and keep workloads from being placed in unhealthy or unreachable clusters, we may consider the similar logic of taint/toleration in Kubernetes. Basically, there are two things we could do:
+
+1) Implement taint/toleration in placement apis.
+2) Add a new controller to map states of clusters to taints automatically.
+
+Taints and tolerations would work together to allow users to control the selection of managed clusters more flexibly, and evict selected clusters after a certain period. Specifically, Taints are properties of managed clusters, they allow a placement to repel a set of managed clusters. Tolerations are applied to placements, and allow the managed clusters with matching taints to be scheduled onto placements.
+
+### User Stories
+
+Story 1: Users can prevent some workloads from being scheduled to those managed clusters.
```

Comment:
The true scenario of story1 and story 4 I think includes:
User can set a  cluster to maintain status so workload will be evicted and no workload  will be scheduled to a cluster. User can set security configuration still in these clusters even the cluster is in maintain mode.